pkg/account/persistence/loader/transform.go

```diff
@@ -21,30 +21,17 @@ import (
 	persistence "github.com/dynatrace/dynatrace-configuration-as-code/v2/pkg/account/persistence/internal/types"
 )
 
-func transformToAccountResources(resources *persistence.Resources) *account.Resources {
-	return &account.Resources{
-		Policies:     transformPolicies(resources.Policies),
-		Groups:       transformGroups(resources.Groups),
-		Users:        transformUsers(resources.Users),
-		ServiceUsers: transformServiceUsers(resources.ServiceUsers),
+func transformPolicy(pPolicy persistence.Policy) account.Policy {
```

Comment:
And then, looking from this angle, I guess I would prefer a `policy` package with all the code in it. And if so we would do it in a refactoring.